aws/users/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "aws_assume_role_arn" {}
+
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `cp` or `cloudposse`)"
+}
+
+variable "stage" {
+  type        = "string"
+  description = "Stage (e.g. `prod`, `dev`, `staging`)"
+}
+
+variable "name" {
+  type        = "string"
+  description = "Application or solution name (e.g. `app`)"
+  default     = "terraform"
+}
+
+variable "smtp_username" {
```

Comment:
looks like all vars below are not in use

Reply:
See https://github.com/cloudposse/root.cloudposse.co/pull/24

Reply:
ok I see